Extension/src/LanguageServer/configurations.ts

```diff
@@ -832,7 +842,7 @@ export class CppProperties {
                         }
                     }
                     let diagnostic: vscode.Diagnostic = new vscode.Diagnostic(
-                        new vscode.Range(document.positionAt(curTextStartOffset + curOffset + 1), document.positionAt(curTextStartOffset + curOffset + curPath.length - 1)),
+                        new vscode.Range(document.positionAt(curTextStartOffset + curOffset + 1), document.positionAt(curTextStartOffset + curOffset + (curPath.endsWith("\"") ? curPath.length - 1 : curPath.length))),
```

Comment:
Btw, you can embed quotes in a string in JavaScript, as the begin quote type will match to the end quote type. I.e. you can write: `curPath.endsWith('"')`

Reply:
Yeah, I used '"' a few lines above. I can change it to that this case.